### PR TITLE
Add redirect/email verify logic for `in-bundle`

### PR DESCRIPTION
**Pull request by williamchong007**

### CODE REVIEW

File: store/modules/actions/user.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ export async function getWalletByUser({ commit }, id) {
 export async function sendVerifyEmail({ commit, rootState }, { id, ref }) {
   /* guard ref to only 'tokensale' for now */
```
Comment (nwingt) [inline store/modules/actions/user.js]: Can remove this comment?

File: components/checkoutForm.vue [hunk 2]
```diff
@@ -40,6 +40,7 @@
         </li>
       </ul>
     </div>
+    <email-dialog ref="emailDialog" :emailRef="'in-bundle'" />
```
Comment (nwingt) [inline components/checkoutForm.vue]: I think it is better to place the email dialog inside bundle page, as the child component should know nothing about its container

Comment (williamchong007) [reply components/checkoutForm.vue]: then the email checking logic (or even the checkout logic?) should be moved too?
or just fire event from inside and trigger `email-dialog` accordingly ?

Comment (nwingt) [reply components/checkoutForm.vue]: Maybe pass `emailRef` as props first

**Pull request merged**